[google-cloud-cpp] Build failure on arm64-osx

Package: google-cloud-cpp[core,rest-common,sql]:arm64-osx -> 2.19.0

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 15.0.0.15000040
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 72010900b 2023-12-04 (19 hours ago)

**To Reproduce**

`vcpkg x-set-installed --enforce-port-checks --allow-unsupported google-cloud-cpp[core,sql]`
**Failure logs**

```
-- Downloading https://github.com/googleapis/google-cloud-cpp/archive/v2.19.0.tar.gz -> googleapis-google-cloud-cpp-v2.19.0.tar.gz...
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/googleapis-google-cloud-cpp-v2.19.0.tar.gz
-- Applying patch support_absl_cxx17.patch
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.19.0-b6676b95f0.clean
-- Found external ninja('1.11.1').
-- Configuring arm64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: /opt/homebrew/Cellar/cmake/3.27.9/bin/cmake /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.19.0-b6676b95f0.clean -G Ninja -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=/Users/leanderSchulten/git_projekte/vcpkg/packages/google-cloud-cpp_arm64-osx/debug -DFETCHCONTENT_FULLY_DISCONNECTED=ON -DGOOGLE_CLOUD_CPP_ENABLE=sql -DGOOGLE_CLOUD_CPP_ENABLE_MACOS_OPENSSL_CHECK=OFF -DGOOGLE_CLOUD_CPP_ENABLE_WERROR=OFF -DGOOGLE_CLOUD_CPP_ENABLE_CCACHE=OFF -DGOOGLE_CLOUD_CPP_ENABLE_EXAMPLES=OFF -DBUILD_TESTING=OFF -DGOOGLE_CLOUD_CPP_ENABLE_CTYPE_CORD_WORKAROUND=ON -DCMAKE_MAKE_PROGRAM=/opt/homebrew/bin/ninja -DCMAKE_SYSTEM_NAME=Darwin -DBUILD_SHARED_LIBS=OFF -DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/Users/leanderSchulten/git_projekte/vcpkg/scripts/toolchains/osx.cmake -DVCPKG_TARGET_TRIPLET=arm64-osx -DVCPKG_SET_CHARSET_FLAG=ON -DVCPKG_PLATFORM_TOOLSET=external -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON -DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON -DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE -DCMAKE_VERBOSE_MAKEFILE=ON -DVCPKG_APPLOCAL_DEPS=OFF -DCMAKE_TOOLCHAIN_FILE=/Users/leanderSchulten/git_projekte/vcpkg/scripts/buildsystems/vcpkg.cmake -DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON -DVCPKG_CXX_FLAGS= -DVCPKG_CXX_FLAGS_RELEASE= -DVCPKG_CXX_FLAGS_DEBUG= -DVCPKG_C_FLAGS= -DVCPKG_C_FLAGS_RELEASE= -DVCPKG_C_FLAGS_DEBUG= -DVCPKG_CRT_LINKAGE=dynamic -DVCPKG_LINKER_FLAGS= -DVCPKG_LINKER_FLAGS_RELEASE= -DVCPKG_LINKER_FLAGS_DEBUG= -DVCPKG_TARGET_ARCHITECTURE=arm64 -DCMAKE_INSTALL_LIBDIR:STRING=lib -DCMAKE_INSTALL_BINDIR:STRING=bin -D_VCPKG_ROOT_DIR=/Users/leanderSchulten/git_projekte/vcpkg -D_VCPKG_INSTALLED_DIR=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed -DVCPKG_MANIFEST_INSTALL=OFF -DCMAKE_OSX_ARCHITECTURES=arm64
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg
    Error code: 1
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/config-arm64-osx-dbg-CMakeCache.txt.log
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/config-arm64-osx-dbg-out.log
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/config-arm64-osx-dbg-err.log

Call Stack (most recent call first):
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:267 (vcpkg_execute_required_process)
  ports/google-cloud-cpp/portfile.cmake:37 (vcpkg_cmake_configure)
  scripts/ports.cmake:170 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/config-arm64-osx-dbg-out.log</summary>

```
-- The CXX compiler identification is AppleClang 15.0.0.15000040
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Could NOT find Protobuf (missing: Protobuf_LIBRARIES Protobuf_INCLUDE_DIR) 
-- Configuring incomplete, errors occurred!
```
</details>

<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/config-arm64-osx-dbg-err.log</summary>

```
CMake Error at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/CMakeFiles/CMakeTmp/CMakeLists.txt:20 (target_link_libraries):
  Target "cmTC_ccde9" links to:

    gRPC::grpc++

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



CMake Error at cmake/FindgRPC.cmake:283 (try_run):
  Failed to generate test project build system.
Call Stack (most recent call first):
  /Users/leanderSchulten/git_projekte/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package)
  CMakeLists.txt:201 (find_package)
```
</details>

<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/config-arm64-osx-dbg-CMakeCache.txt.log</summary>

```
# This is the CMakeCache file.
# For build in directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg
# It was generated by CMake: /opt/homebrew/Cellar/cmake/3.27.9/bin/cmake
# You can edit this file to change values found and used by cmake.
# If you do not want to change any of the values, simply exit the editor.
# If you do want to change a value, simply edit, save, and exit the editor.
# The syntax for the file is as follows:
# KEY:TYPE=VALUE
# KEY is the name of a variable in the cache.
# TYPE is a hint to GUIs for the type of VALUE, DO NOT EDIT TYPE!.
# VALUE is the current value for the KEY.

########################
# EXTERNAL cache entries
########################

//No help, variable specified on the command line.
BUILD_SHARED_LIBS:UNINITIALIZED=OFF

//No help, variable specified on the command line.
BUILD_TESTING:UNINITIALIZED=OFF

//Path to a program.
CMAKE_ADDR2LINE:FILEPATH=CMAKE_ADDR2LINE-NOTFOUND

//Path to a program.
CMAKE_AR:FILEPATH=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ar

//Choose the type of build, options are: None Debug Release RelWithDebInfo
// MinSizeRel ...
CMAKE_BUILD_TYPE:STRING=Debug

CMAKE_CROSSCOMPILING:STRING=OFF

//CXX compiler
CMAKE_CXX_COMPILER:FILEPATH=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++

//Compiler launcher for CXX.
CMAKE_CXX_COMPILER_LAUNCHER:STRING=ccache

//Flags used by the CXX compiler during all build types.
CMAKE_CXX_FLAGS:STRING=-fPIC

//Flags used by the CXX compiler during DEBUG builds.
CMAKE_CXX_FLAGS_DEBUG:STRING=-g

//Flags used by the CXX compiler during MINSIZEREL builds.
CMAKE_CXX_FLAGS_MINSIZEREL:STRING=-Os -DNDEBUG

//Flags used by the CXX compiler during RELEASE builds.
CMAKE_CXX_FLAGS_RELEASE:STRING=-O3 -DNDEBUG

//Flags used by the CXX compiler during RELWITHDEBINFO builds.
CMAKE_CXX_FLAGS_RELWITHDEBINFO:STRING=-O2 -g -DNDEBUG

//Path to a program.
CMAKE_DLLTOOL:FILEPATH=CMAKE_DLLTOOL-NOTFOUND

//No help, variable specified on the command line.
CMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION:UNINITIALIZED=ON

//Flags used by the linker during all build types.
CMAKE_EXE_LINKER_FLAGS:STRING=

//Flags used by the linker during DEBUG builds.
CMAKE_EXE_LINKER_FLAGS_DEBUG:STRING=

//Flags used by the linker during MINSIZEREL builds.
CMAKE_EXE_LINKER_FLAGS_MINSIZEREL:STRING=

//Flags used by the linker during RELEASE builds.
CMAKE_EXE_LINKER_FLAGS_RELEASE:STRING=

//Flags used by the linker during RELWITHDEBINFO builds.
CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO:STRING=

//Enable/Disable output of compile commands during generation.
CMAKE_EXPORT_COMPILE_COMMANDS:BOOL=

//No help, variable specified on the command line.
CMAKE_EXPORT_NO_PACKAGE_REGISTRY:UNINITIALIZED=ON

//No help, variable specified on the command line.
CMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY:UNINITIALIZED=ON

//No help, variable specified on the command line.
CMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY:UNINITIALIZED=ON

//Value Computed by CMake.
CMAKE_FIND_PACKAGE_REDIRECTS_DIR:STATIC=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/CMakeFiles/pkgRedirects

//No help, variable specified on the command line.
CMAKE_INSTALL_BINDIR:STRING=bin

//No help, variable specified on the command line.
CMAKE_INSTALL_LIBDIR:STRING=lib

//Path to a program.
CMAKE_INSTALL_NAME_TOOL:FILEPATH=/usr/bin/install_name_tool

//Install path prefix, prepended onto install directories.
CMAKE_INSTALL_PREFIX:PATH=/Users/leanderSchulten/git_projekte/vcpkg/packages/google-cloud-cpp_arm64-osx/debug

//No help, variable specified on the command line.
CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP:UNINITIALIZED=TRUE

//Path to a program.
CMAKE_LINKER:FILEPATH=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld

CMAKE_MACOSX_RPATH:BOOL=ON

//No help, variable specified on the command line.
CMAKE_MAKE_PROGRAM:UNINITIALIZED=/opt/homebrew/bin/ninja

//Flags used by the linker during the creation of modules during
// all build types.
CMAKE_MODULE_LINKER_FLAGS:STRING=

//Flags used by the linker during the creation of modules during
// DEBUG builds.
CMAKE_MODULE_LINKER_FLAGS_DEBUG:STRING=

//Flags used by the linker during the creation of modules during
// MINSIZEREL builds.
CMAKE_MODULE_LINKER_FLAGS_MINSIZEREL:STRING=

//Flags used by the linker during the creation of modules during
// RELEASE builds.
CMAKE_MODULE_LINKER_FLAGS_RELEASE:STRING=

//Flags used by the linker during the creation of modules during
// RELWITHDEBINFO builds.
CMAKE_MODULE_LINKER_FLAGS_RELWITHDEBINFO:STRING=

//Path to a program.
CMAKE_NM:FILEPATH=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/nm

//Path to a program.
CMAKE_OBJCOPY:FILEPATH=CMAKE_OBJCOPY-NOTFOUND

//Path to a program.
CMAKE_OBJDUMP:FILEPATH=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/objdump

//Build architectures for OSX
CMAKE_OSX_ARCHITECTURES:STRING=arm64

//Minimum OS X version to target for deployment (at runtime); newer
// APIs weak linked. Set to empty string for default value.
CMAKE_OSX_DEPLOYMENT_TARGET:STRING=13.6

//The product will be built against the headers and libraries located
// inside the indicated SDK.
CMAKE_OSX_SYSROOT:PATH=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk

//Value Computed by CMake
CMAKE_PROJECT_DESCRIPTION:STATIC=

//Value Computed by CMake
CMAKE_PROJECT_HOMEPAGE_URL:STATIC=

//Value Computed by CMake
CMAKE_PROJECT_NAME:STATIC=google-cloud-cpp

//Value Computed by CMake
CMAKE_PROJECT_VERSION:STATIC=2.19.0

//Value Computed by CMake
CMAKE_PROJECT_VERSION_MAJOR:STATIC=2

//Value Computed by CMake
CMAKE_PROJECT_VERSION_MINOR:STATIC=19

//Value Computed by CMake
CMAKE_PROJECT_VERSION_PATCH:STATIC=0

//Value Computed by CMake
CMAKE_PROJECT_VERSION_TWEAK:STATIC=

//Path to a program.
CMAKE_RANLIB:FILEPATH=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib

//Path to a program.
CMAKE_READELF:FILEPATH=CMAKE_READELF-NOTFOUND

//Flags used by the linker during the creation of shared libraries
// during all build types.
CMAKE_SHARED_LINKER_FLAGS:STRING=

//Flags used by the linker during the creation of shared libraries
// during DEBUG builds.
CMAKE_SHARED_LINKER_FLAGS_DEBUG:STRING=

//Flags used by the linker during the creation of shared libraries
// during MINSIZEREL builds.
CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL:STRING=

//Flags used by the linker during the creation of shared libraries
// during RELEASE builds.
CMAKE_SHARED_LINKER_FLAGS_RELEASE:STRING=

//Flags used by the linker during the creation of shared libraries
// during RELWITHDEBINFO builds.
CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO:STRING=

//If set, runtime paths are not added when installing shared libraries,
// but are added when building.
CMAKE_SKIP_INSTALL_RPATH:BOOL=NO

//If set, runtime paths are not added when using shared libraries.
CMAKE_SKIP_RPATH:BOOL=NO

//Flags used by the linker during the creation of static libraries
// during all build types.
CMAKE_STATIC_LINKER_FLAGS:STRING=

//Flags used by the linker during the creation of static libraries
// during DEBUG builds.
CMAKE_STATIC_LINKER_FLAGS_DEBUG:STRING=

//Flags used by the linker during the creation of static libraries
// during MINSIZEREL builds.
CMAKE_STATIC_LINKER_FLAGS_MINSIZEREL:STRING=

//Flags used by the linker during the creation of static libraries
// during RELEASE builds.
CMAKE_STATIC_LINKER_FLAGS_RELEASE:STRING=

//Flags used by the linker during the creation of static libraries
// during RELWITHDEBINFO builds.
CMAKE_STATIC_LINKER_FLAGS_RELWITHDEBINFO:STRING=

//Path to a program.
CMAKE_STRIP:FILEPATH=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/strip

CMAKE_SYSTEM_NAME:STRING=Darwin

CMAKE_SYSTEM_PROCESSOR:STRING=arm64

CMAKE_SYSTEM_VERSION:STRING=22.6.0

//Path to a program.
CMAKE_TAPI:FILEPATH=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/tapi

//The CMake toolchain file
CMAKE_TOOLCHAIN_FILE:FILEPATH=/Users/leanderSchulten/git_projekte/vcpkg/scripts/buildsystems/vcpkg.cmake

//If this value is on, makefiles will be generated without the
// .SILENT directive, and all commands will be echoed to the console
// during the make.  This is useful for debugging only. With Visual
// Studio IDE projects all commands are done without /nologo.
CMAKE_VERBOSE_MAKEFILE:BOOL=ON

//No help, variable specified on the command line.
FETCHCONTENT_FULLY_DISCONNECTED:UNINITIALIZED=ON

//The list of libraries to build.
GOOGLE_CLOUD_CPP_ENABLE:STRING=sql

//Enable building the Bigquery library. Deprecated, prefer GOOGLE_CLOUD_CPP_ENABLE.
GOOGLE_CLOUD_CPP_ENABLE_BIGQUERY:BOOL=ON

//Enable building the Bigtable library. Deprecated, prefer GOOGLE_CLOUD_CPP_ENABLE.
GOOGLE_CLOUD_CPP_ENABLE_BIGTABLE:BOOL=ON

//Automatically use ccache if available
GOOGLE_CLOUD_CPP_ENABLE_CCACHE:BOOL=OFF

//Enable the workarounds for [ctype = CORD] when using Protobuf
// < v23.
//\nMore details at
//\n
//\nhttps://github.com/googleapis/google-cloud-cpp/blob/main/doc/ctype-cord-workarounds.md
//\n
GOOGLE_CLOUD_CPP_ENABLE_CTYPE_CORD_WORKAROUND:BOOL=ON

//No help, variable specified on the command line.
GOOGLE_CLOUD_CPP_ENABLE_EXAMPLES:UNINITIALIZED=OFF

//Enable building the generator. Deprecated, prefer GOOGLE_CLOUD_CPP_ENABLE.
GOOGLE_CLOUD_CPP_ENABLE_GENERATOR:BOOL=OFF

//Enable building the gRPC utilities library. This is now unused,
// the grpc_utils library is only enabled if GOOGLE_CLOUD_CPP_ENABLE
// contains any feature that requires it.
GOOGLE_CLOUD_CPP_ENABLE_GRPC:BOOL=OFF

//Enable building the IAM library. Deprecated, prefer GOOGLE_CLOUD_CPP_ENABLE.
GOOGLE_CLOUD_CPP_ENABLE_IAM:BOOL=ON

//Enable building the Logging library. Deprecated, prefer GOOGLE_CLOUD_CPP_ENABLE.
GOOGLE_CLOUD_CPP_ENABLE_LOGGING:BOOL=ON

//If enabled, check that the user has defined OPENSSL_ROOT_DIR
// on macOS
GOOGLE_CLOUD_CPP_ENABLE_MACOS_OPENSSL_CHECK:BOOL=OFF

//Enable building the Pub/Sub library. Deprecated, prefer GOOGLE_CLOUD_CPP_ENABLE.
GOOGLE_CLOUD_CPP_ENABLE_PUBSUB:BOOL=ON

//Enable building the REST transport library. This is now unused,
// the rest_internal library is only enabled if GOOGLE_CLOUD_CPP_ENABLE
// contains any feature that requires it.
GOOGLE_CLOUD_CPP_ENABLE_REST:BOOL=OFF

//Enable building the Spanner library. Deprecated, prefer GOOGLE_CLOUD_CPP_ENABLE.
GOOGLE_CLOUD_CPP_ENABLE_SPANNER:BOOL=ON

//Enable building the Storage library. Deprecated, prefer GOOGLE_CLOUD_CPP_ENABLE.
GOOGLE_CLOUD_CPP_ENABLE_STORAGE:BOOL=ON

//No help, variable specified on the command line.
GOOGLE_CLOUD_CPP_ENABLE_WERROR:UNINITIALIZED=OFF

//Generate Doxygen documentation. Only google-cloud-cpp developers
// are expected to
//\nuse this flag.
GOOGLE_CLOUD_CPP_GENERATE_DOXYGEN:BOOL=OFF

//Compile the docfx tool. Only google-cloud-cpp developers are
// expected to use this flag.
GOOGLE_CLOUD_CPP_INTERNAL_DOCFX:BOOL=OFF

//Override the default URL for the googleapis protos. google-cloud-cpp
// needs the
//\nproto and gRPC definitions for most Google Cloud services. By
// default these
//\ndefinitions are downloaded from GitHub. In some environments,
// this may not be
//\npossible. This variable can be used to use a local directory,
// or a local
//\ntarball, with the proto definitions.
//\n
//\nIf you override this variable, you must ensure the proto definitions
// match the
//\nexpectations in google-cloud-cpp. Using the same SHA is the
// recommended
//\npractice. Using older SHAs is **NOT** supported, and will almost
// certainly
//\nresult in build problems. You may be able to use a more recent
// SHA of the
//\nprotos, but do not expect that new features will become exposed
// via the C++
//\nclient libraries.
//\n
GOOGLE_CLOUD_CPP_OVERRIDE_GOOGLEAPIS_URL:STRING=

//Override the hash for the googleapis protos tarball. See
//\nGOOGLE_CLOUD_CPP_OVERRIDE_GOOGLEAPIS_URL for details.
//\n
GOOGLE_CLOUD_CPP_OVERRIDE_GOOGLEAPIS_URL_HASH:STRING=

//Enable compilation for the GCS gRPC plugin (EXPERIMENTAL).  Deprecated,
// prefer GOOGLE_CLOUD_CPP_ENABLE.
GOOGLE_CLOUD_CPP_STORAGE_ENABLE_GRPC:BOOL=OFF

//Use a pre-installed version of the common libraries and proto
// files.
//\n
//\nThis option is required to support package maintainers that
// split
//\n`google-cloud-cpp` into multiple packages.
//\n
GOOGLE_CLOUD_CPP_USE_INSTALLED_COMMON:BOOL=OFF

//Link to "latest" in our reference documentation. Otherwise, link
// to the project
//\nversion. Only google-cloud-cpp developers are expected to use
// this flag.
GOOGLE_CLOUD_CPP_USE_LATEST_FOR_REFDOC_LINKS:BOOL=OFF

//The directory containing a CMake configuration file for Protobuf.
Protobuf_DIR:PATH=Protobuf_DIR-NOTFOUND

//Path to a file.
Protobuf_INCLUDE_DIR:PATH=Protobuf_INCLUDE_DIR-NOTFOUND

//Path to a library.
Protobuf_LIBRARY_DEBUG:FILEPATH=Protobuf_LIBRARY_DEBUG-NOTFOUND

//Path to a library.
Protobuf_LIBRARY_RELEASE:FILEPATH=Protobuf_LIBRARY_RELEASE-NOTFOUND

//Path to a library.
Protobuf_LITE_LIBRARY_DEBUG:FILEPATH=Protobuf_LITE_LIBRARY_DEBUG-NOTFOUND

//Path to a library.
Protobuf_LITE_LIBRARY_RELEASE:FILEPATH=Protobuf_LITE_LIBRARY_RELEASE-NOTFOUND

//The Google Protocol Buffers Compiler
Protobuf_PROTOC_EXECUTABLE:FILEPATH=Protobuf_PROTOC_EXECUTABLE-NOTFOUND

//Path to a library.
Protobuf_PROTOC_LIBRARY_DEBUG:FILEPATH=Protobuf_PROTOC_LIBRARY_DEBUG-NOTFOUND

//Path to a library.
Protobuf_PROTOC_LIBRARY_RELEASE:FILEPATH=Protobuf_PROTOC_LIBRARY_RELEASE-NOTFOUND

//Automatically copy dependencies into the output directory for
// executables.
VCPKG_APPLOCAL_DEPS:BOOL=OFF

//No help, variable specified on the command line.
VCPKG_CHAINLOAD_TOOLCHAIN_FILE:UNINITIALIZED=/Users/leanderSchulten/git_projekte/vcpkg/scripts/toolchains/osx.cmake

//No help, variable specified on the command line.
VCPKG_CRT_LINKAGE:UNINITIALIZED=dynamic

//No help, variable specified on the command line.
VCPKG_CXX_FLAGS:UNINITIALIZED=

//No help, variable specified on the command line.
VCPKG_CXX_FLAGS_DEBUG:UNINITIALIZED=

//No help, variable specified on the command line.
VCPKG_CXX_FLAGS_RELEASE:UNINITIALIZED=

//No help, variable specified on the command line.
VCPKG_C_FLAGS:UNINITIALIZED=

//No help, variable specified on the command line.
VCPKG_C_FLAGS_DEBUG:UNINITIALIZED=

//No help, variable specified on the command line.
VCPKG_C_FLAGS_RELEASE:UNINITIALIZED=

//The directory which contains the installed libraries for each
// triplet
VCPKG_INSTALLED_DIR:PATH=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed

//No help, variable specified on the command line.
VCPKG_LINKER_FLAGS:UNINITIALIZED=

//No help, variable specified on the command line.
VCPKG_LINKER_FLAGS_DEBUG:UNINITIALIZED=

//No help, variable specified on the command line.
VCPKG_LINKER_FLAGS_RELEASE:UNINITIALIZED=

//The path to the vcpkg manifest directory.
VCPKG_MANIFEST_DIR:PATH=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.19.0-b6676b95f0.clean

//Install the dependencies listed in your manifest:
//\n    If this is off, you will have to manually install your dependencies.
//\n    See https://github.com/microsoft/vcpkg/tree/master/docs/specifications/manifests.md
// for more info.
//\n
VCPKG_MANIFEST_INSTALL:BOOL=OFF

//Use manifest mode, as opposed to classic mode.
VCPKG_MANIFEST_MODE:BOOL=ON

//No help, variable specified on the command line.
VCPKG_PLATFORM_TOOLSET:UNINITIALIZED=external

//Appends the vcpkg paths to CMAKE_PREFIX_PATH, CMAKE_LIBRARY_PATH
// and CMAKE_FIND_ROOT_PATH so that vcpkg libraries/packages are
// found after toolchain/system libraries/packages.
VCPKG_PREFER_SYSTEM_LIBS:BOOL=OFF

//Enable the setup of CMAKE_PROGRAM_PATH to vcpkg paths
VCPKG_SETUP_CMAKE_PROGRAM_PATH:BOOL=ON

//No help, variable specified on the command line.
VCPKG_SET_CHARSET_FLAG:UNINITIALIZED=ON

//No help, variable specified on the command line.
VCPKG_TARGET_ARCHITECTURE:UNINITIALIZED=arm64

//Vcpkg target triplet (ex. x86-windows)
VCPKG_TARGET_TRIPLET:STRING=arm64-osx

//Trace calls to find_package()
VCPKG_TRACE_FIND_PACKAGE:BOOL=OFF

//Enables messages from the VCPKG toolchain for debugging purposes.
VCPKG_VERBOSE:BOOL=OFF

//(experimental) Automatically copy dependencies into the install
// target directory for executables. Requires CMake 3.14.
X_VCPKG_APPLOCAL_DEPS_INSTALL:BOOL=OFF

//(experimental) Add USES_TERMINAL to VCPKG_APPLOCAL_DEPS to force
// serialization.
X_VCPKG_APPLOCAL_DEPS_SERIALIZED:BOOL=OFF

//The directory which contains the installed libraries for each
// triplet
_VCPKG_INSTALLED_DIR:PATH=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed

//No help, variable specified on the command line.
_VCPKG_ROOT_DIR:UNINITIALIZED=/Users/leanderSchulten/git_projekte/vcpkg

//Path to a file.
_gRPC_INCLUDE_DIR:PATH=_gRPC_INCLUDE_DIR-NOTFOUND

//Path to a library.
_gRPC_grpc++_LIBRARY_DEBUG:FILEPATH=_gRPC_grpc++_LIBRARY_DEBUG-NOTFOUND

//Path to a library.
_gRPC_grpc++_LIBRARY_RELEASE:FILEPATH=_gRPC_grpc++_LIBRARY_RELEASE-NOTFOUND

//Path to a library.
_gRPC_grpc_LIBRARY_DEBUG:FILEPATH=_gRPC_grpc_LIBRARY_DEBUG-NOTFOUND

//Path to a library.
_gRPC_grpc_LIBRARY_RELEASE:FILEPATH=_gRPC_grpc_LIBRARY_RELEASE-NOTFOUND

//The directory containing a CMake configuration file for absl.
absl_DIR:PATH=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/absl

//The directory containing a CMake configuration file for gRPC.
gRPC_DIR:PATH=gRPC_DIR-NOTFOUND

//Value Computed by CMake
google-cloud-cpp_BINARY_DIR:STATIC=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg

//Value Computed by CMake
google-cloud-cpp_IS_TOP_LEVEL:STATIC=ON

//Value Computed by CMake
google-cloud-cpp_SOURCE_DIR:STATIC=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.19.0-b6676b95f0.clean


########################
# INTERNAL cache entries
########################

//ADVANCED property for variable: CMAKE_ADDR2LINE
CMAKE_ADDR2LINE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_AR
CMAKE_AR-ADVANCED:INTERNAL=1
//This is the directory where this CMakeCache.txt was created
CMAKE_CACHEFILE_DIR:INTERNAL=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg
//Major version of cmake used to create the current loaded cache
CMAKE_CACHE_MAJOR_VERSION:INTERNAL=3
//Minor version of cmake used to create the current loaded cache
CMAKE_CACHE_MINOR_VERSION:INTERNAL=27
//Patch version of cmake used to create the current loaded cache
CMAKE_CACHE_PATCH_VERSION:INTERNAL=9
//Path to CMake executable.
CMAKE_COMMAND:INTERNAL=/opt/homebrew/Cellar/cmake/3.27.9/bin/cmake
//Path to cpack program executable.
CMAKE_CPACK_COMMAND:INTERNAL=/opt/homebrew/Cellar/cmake/3.27.9/bin/cpack
//Path to ctest program executable.
CMAKE_CTEST_COMMAND:INTERNAL=/opt/homebrew/Cellar/cmake/3.27.9/bin/ctest
//ADVANCED property for variable: CMAKE_CXX_COMPILER
CMAKE_CXX_COMPILER-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_FLAGS
CMAKE_CXX_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_FLAGS_DEBUG
CMAKE_CXX_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_FLAGS_MINSIZEREL
CMAKE_CXX_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_FLAGS_RELEASE
CMAKE_CXX_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_FLAGS_RELWITHDEBINFO
CMAKE_CXX_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_DLLTOOL
CMAKE_DLLTOOL-ADVANCED:INTERNAL=1
//Path to cache edit program executable.
CMAKE_EDIT_COMMAND:INTERNAL=/opt/homebrew/Cellar/cmake/3.27.9/bin/ccmake
//Executable file format
CMAKE_EXECUTABLE_FORMAT:INTERNAL=MACHO
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS
CMAKE_EXE_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_DEBUG
CMAKE_EXE_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_MINSIZEREL
CMAKE_EXE_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_RELEASE
CMAKE_EXE_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXPORT_COMPILE_COMMANDS
CMAKE_EXPORT_COMPILE_COMMANDS-ADVANCED:INTERNAL=1
//Name of external makefile project generator.
CMAKE_EXTRA_GENERATOR:INTERNAL=
//Name of generator.
CMAKE_GENERATOR:INTERNAL=Ninja
//Generator instance identifier.
CMAKE_GENERATOR_INSTANCE:INTERNAL=
//Name of generator platform.
CMAKE_GENERATOR_PLATFORM:INTERNAL=
//Name of generator toolset.
CMAKE_GENERATOR_TOOLSET:INTERNAL=
//Test CMAKE_HAVE_LIBC_PTHREAD
CMAKE_HAVE_LIBC_PTHREAD:INTERNAL=1
//Source directory with the top level CMakeLists.txt file for this
// project
CMAKE_HOME_DIRECTORY:INTERNAL=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.19.0-b6676b95f0.clean
//ADVANCED property for variable: CMAKE_INSTALL_NAME_TOOL
CMAKE_INSTALL_NAME_TOOL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_LINKER
CMAKE_LINKER-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS
CMAKE_MODULE_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_DEBUG
CMAKE_MODULE_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_MINSIZEREL
CMAKE_MODULE_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_RELEASE
CMAKE_MODULE_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_MODULE_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_NM
CMAKE_NM-ADVANCED:INTERNAL=1
//number of local generators
CMAKE_NUMBER_OF_MAKEFILES:INTERNAL=1
//ADVANCED property for variable: CMAKE_OBJCOPY
CMAKE_OBJCOPY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_OBJDUMP
CMAKE_OBJDUMP-ADVANCED:INTERNAL=1
//Platform information initialized
CMAKE_PLATFORM_INFO_INITIALIZED:INTERNAL=1
//ADVANCED property for variable: CMAKE_RANLIB
CMAKE_RANLIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_READELF
CMAKE_READELF-ADVANCED:INTERNAL=1
//Path to CMake installation.
CMAKE_ROOT:INTERNAL=/opt/homebrew/Cellar/cmake/3.27.9/share/cmake
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS
CMAKE_SHARED_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_DEBUG
CMAKE_SHARED_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL
CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_RELEASE
CMAKE_SHARED_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SKIP_INSTALL_RPATH
CMAKE_SKIP_INSTALL_RPATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SKIP_RPATH
CMAKE_SKIP_RPATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS
CMAKE_STATIC_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_DEBUG
CMAKE_STATIC_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_MINSIZEREL
CMAKE_STATIC_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_RELEASE
CMAKE_STATIC_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_STATIC_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STRIP
CMAKE_STRIP-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_TAPI
CMAKE_TAPI-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_TOOLCHAIN_FILE
CMAKE_TOOLCHAIN_FILE-ADVANCED:INTERNAL=1
//uname command
CMAKE_UNAME:INTERNAL=/usr/bin/uname
//ADVANCED property for variable: CMAKE_VERBOSE_MAKEFILE
CMAKE_VERBOSE_MAKEFILE-ADVANCED:INTERNAL=1
//Details about finding Threads
FIND_PACKAGE_MESSAGE_DETAILS_Threads:INTERNAL=[TRUE][v()]
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_ENABLE_BIGQUERY
GOOGLE_CLOUD_CPP_ENABLE_BIGQUERY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_ENABLE_BIGTABLE
GOOGLE_CLOUD_CPP_ENABLE_BIGTABLE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_ENABLE_CCACHE
GOOGLE_CLOUD_CPP_ENABLE_CCACHE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_ENABLE_CTYPE_CORD_WORKAROUND
GOOGLE_CLOUD_CPP_ENABLE_CTYPE_CORD_WORKAROUND-ADVANCED:INTERNAL=1
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_ENABLE_GENERATOR
GOOGLE_CLOUD_CPP_ENABLE_GENERATOR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_ENABLE_GRPC
GOOGLE_CLOUD_CPP_ENABLE_GRPC-ADVANCED:INTERNAL=1
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_ENABLE_IAM
GOOGLE_CLOUD_CPP_ENABLE_IAM-ADVANCED:INTERNAL=1
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_ENABLE_LOGGING
GOOGLE_CLOUD_CPP_ENABLE_LOGGING-ADVANCED:INTERNAL=1
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_ENABLE_PUBSUB
GOOGLE_CLOUD_CPP_ENABLE_PUBSUB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_ENABLE_REST
GOOGLE_CLOUD_CPP_ENABLE_REST-ADVANCED:INTERNAL=1
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_ENABLE_SPANNER
GOOGLE_CLOUD_CPP_ENABLE_SPANNER-ADVANCED:INTERNAL=1
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_ENABLE_STORAGE
GOOGLE_CLOUD_CPP_ENABLE_STORAGE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_GENERATE_DOXYGEN
GOOGLE_CLOUD_CPP_GENERATE_DOXYGEN-ADVANCED:INTERNAL=1
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_INTERNAL_DOCFX
GOOGLE_CLOUD_CPP_INTERNAL_DOCFX-ADVANCED:INTERNAL=1
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_OVERRIDE_GOOGLEAPIS_URL
GOOGLE_CLOUD_CPP_OVERRIDE_GOOGLEAPIS_URL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_OVERRIDE_GOOGLEAPIS_URL_HASH
GOOGLE_CLOUD_CPP_OVERRIDE_GOOGLEAPIS_URL_HASH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_STORAGE_ENABLE_GRPC
GOOGLE_CLOUD_CPP_STORAGE_ENABLE_GRPC-ADVANCED:INTERNAL=1
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_USE_INSTALLED_COMMON
GOOGLE_CLOUD_CPP_USE_INSTALLED_COMMON-ADVANCED:INTERNAL=1
//ADVANCED property for variable: GOOGLE_CLOUD_CPP_USE_LATEST_FOR_REFDOC_LINKS
GOOGLE_CLOUD_CPP_USE_LATEST_FOR_REFDOC_LINKS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: Protobuf_INCLUDE_DIR
Protobuf_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: Protobuf_LIBRARY_DEBUG
Protobuf_LIBRARY_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: Protobuf_LIBRARY_RELEASE
Protobuf_LIBRARY_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: Protobuf_LITE_LIBRARY_DEBUG
Protobuf_LITE_LIBRARY_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: Protobuf_LITE_LIBRARY_RELEASE
Protobuf_LITE_LIBRARY_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: Protobuf_PROTOC_EXECUTABLE
Protobuf_PROTOC_EXECUTABLE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: Protobuf_PROTOC_LIBRARY_DEBUG
Protobuf_PROTOC_LIBRARY_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: Protobuf_PROTOC_LIBRARY_RELEASE
Protobuf_PROTOC_LIBRARY_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: VCPKG_VERBOSE
VCPKG_VERBOSE-ADVANCED:INTERNAL=1
//Making sure VCPKG_MANIFEST_MODE doesn't change
Z_VCPKG_CHECK_MANIFEST_MODE:INTERNAL=ON
//Vcpkg root directory
Z_VCPKG_ROOT_DIR:INTERNAL=/Users/leanderSchulten/git_projekte/vcpkg
//Result of TRY_COMPILE
_gRPC_GET_VERSION_COMPILE_STATUS:INTERNAL=FALSE
//ADVANCED property for variable: _gRPC_INCLUDE_DIR
_gRPC_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: _gRPC_grpc++_LIBRARY_DEBUG
_gRPC_grpc++_LIBRARY_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: _gRPC_grpc++_LIBRARY_RELEASE
_gRPC_grpc++_LIBRARY_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: _gRPC_grpc_LIBRARY_DEBUG
_gRPC_grpc_LIBRARY_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: _gRPC_grpc_LIBRARY_RELEASE
_gRPC_grpc_LIBRARY_RELEASE-ADVANCED:INTERNAL=1
```
</details>

--- a/ports/google-cloud-cpp/vcpkg.json
+++ b/ports/google-cloud-cpp/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "google-cloud-cpp",
   "version": "2.19.0",
+  "port-version": 1,
   "description": "C++ Client Libraries for Google Cloud Platform APIs.",
   "homepage": "https://github.com/googleapis/google-cloud-cpp",
   "license": "Apache-2.0",
@@ -1247,6 +1248,7 @@
           "name": "google-cloud-cpp",
           "default-features": false,
           "features": [
+            "grpc-common",
             "rest-common"
           ]
         }

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2024,7 +2024,7 @@
       "baseline": "10.1",
       "port-version": 13
     },
-    "cuda-api-wrappers" : {
+    "cuda-api-wrappers": {
       "baseline": "0.6.6",
       "port-version": 0
     },
@@ -3058,7 +3058,7 @@
     },
     "google-cloud-cpp": {
       "baseline": "2.19.0",
-      "port-version": 0
+      "port-version": 1
     },
     "google-cloud-cpp-common": {
       "baseline": "alias",

--- a/versions/g-/google-cloud-cpp.json
+++ b/versions/g-/google-cloud-cpp.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "815f9e2c25c4aa47b16ec69ac6c41e2d52655d89",
+      "version": "2.19.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "cda5a5192b0d1310376b1370890e1d38755c076a",
       "version": "2.19.0",
       "port-version": 0

